src/PharoThings-Devices-BME280.package/PotBME280Device.class/instance/readHumidity.st

```diff
@@ -1,3 +1,3 @@
 controlling
 readHumidity
-	^self readParameters last
\ No newline at end of file
+	^self readParameters last printShowingDecimalPlaces: 1
```

Comment:
I don't think it is good to return string in read methods. Users could implement some logic with those numbers and it should not require parsing numbers from strings.

It would be better to add new methods for that purpose like printHuminity, pringPressure,  printTemperature. It is common practice in smalltalk to use print word in such cases.

Reply:
I see. Which method can I use to format the result to a number with 1 decimal place?

Reply:
Then it is not about formatting but rounding:
```Smalltalk
12.3456 roundTo: 0.01 "==>  12.35"
```

Reply:
Thanks! I found the method round: